indico/modules/events/registration/fields/base.py

```diff
@@ -129,6 +129,10 @@ def view_data(self):
     def get_friendly_data(self, registration_data):
         return registration_data.data
 
+    def get_data_as_short_string(self, registration_data):
```

Comment:
Yes, its clearer that way although we now have a `friendly_data` property and `get_friendly_data` method in the RegistrationData class.